packages/grid/test/column.test.js

```diff
@@ -32,7 +32,8 @@ class GridContainer extends PolymerElement {
           ></vaadin-grid-column>
         </vaadin-grid-column-group>
 
-        <vaadin-grid-column id="emptycolumn"></vaadin-grid-column>
+        <vaadin-grid-column row-header id="emptycolumn"></vaadin-grid-column>
+        <vaadin-grid-column row-header id="rowheadercolumn"></vaadin-grid-column>
```

Comment:
You already define `column - simple grid with rowHeader` for testing the rowHeader functionality. Prefer testing the new feature there in isolation and avoid changing existing tests/test setups unless it's necessary.